vm/systemSmartContracts/eei.go

```diff
@@ -268,10 +268,71 @@ func (host *vmContext) copyFromContext(currContext *vmContext) {
 		}
 	}
 
-	host.outputAccounts = currContext.outputAccounts
+	for _, outAcc := range currContext.outputAccounts {
+		leftAccount, exist := host.outputAccounts[string(outAcc.Address)]
+		if !exist {
+			leftAccount = &vmcommon.OutputAccount{}
+			host.outputAccounts[string(outAcc.Address)] = leftAccount
+		}
+		mergeOutputAccounts(leftAccount, outAcc)
+	}
 	host.scAddress = currContext.scAddress
 }
 
+func mergeOutputAccounts(
```

Comment:
why not move this functionality in vmcommon.OutputAccount.Merge function? It can be used elsewhere and most important, will be kept up-to-date more easily. Otherwise we will add a field in OutputAccount struct and forget to search in the whole project if we have a kind of clone functionality

Reply:
done